src/dlgProfilePreferences.cpp

```diff
@@ -2965,10 +2965,10 @@ void dlgProfilePreferences::populateScriptsList()
     combobox->setUpdatesEnabled(false);
     combobox->clear();
 
-    for (auto item : items) {
-        combobox->addItem(qsl("%1 (%2)").arg(std::get<0>(item), std::get<1>(item)),
+    for (auto [name, type, id] : items) {
+        combobox->addItem(qsl("%1 (%2)").arg(name, type),
                           // store the item type and ID in data so we can pull up the script for it later
-                          QVariant::fromValue(QPair<QString, int>(std::get<1>(item), std::get<2>(item))));
+                          QVariant::fromValue(QPair<QString, int>(type, id)));
```

Comment:
Ironically you also have an alternative to the two element specialism of the `std::tuple` as `std::pair` in the `QPair` Qt equivalent here...! 😜

Reply:
Agreed